android/src/main/java/com/tailscale/ipn/ui/model/NetMap.kt

```diff
@@ -5,51 +5,49 @@ package com.tailscale.ipn.ui.model
 
 import kotlinx.serialization.Serializable
 
-class Netmap {
-    @Serializable
-    data class NetworkMap(
-            var SelfNode: Tailcfg.Node,
-            var NodeKey: KeyNodePublic,
-            var Peers: List<Tailcfg.Node>? = null,
-            var Expiry: Time,
-            var Domain: String,
-            var UserProfiles: Map<String, Tailcfg.UserProfile>,
-            var TKAEnabled: Boolean,
```

Comment:
We'll need some of these fields shortly too.   TAKEnabled before launch.  Domain is used on iOS to disambiguate accounts and tailnet name, etc.